### PR TITLE
Wazo 2257 recording queues groups

### PR DESCRIPTION
**Pull request by pc-m**
Depends-On: https://github.com/wazo-platform/xivo-dao/pull/115

### CODE REVIEW

File: wazo_agid/handlers/groupfeatures.py [hunk 1]
```diff
@@ -142,3 +150,18 @@ def _set_schedule(self):
...
+    def _set_call_record_filename(self):
+        args = {
+            'srcnum': self._agi.get_variable(dialplan_variables.SOURCE_NUMBER),
+            'dstnum': self._exten,
+            'timestamp': int(time.time()),
+            'local_time': time.asctime(time.localtime()),
+            'utc_time': time.asctime(time.gmtime()),
+            'base_context': self._context,
+            'tenant_uuid': self._agi.get_variable(dialplan_variables.TENANT_UUID),
+            'dest_type': 'group',
```
Comment (bloom1) [inline wazo_agid/handlers/groupfeatures.py:163]: I'm not sure I understand why dest_type is group here

**Comment by bloom1** [comment]
recheck

File: wazo_agid/handlers/group.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        if not should_record:
+            return
+
+        calld = self._agi.config['calld']['client']
+        channel_id = self._agi.env['agi_uniqueid']
+        self._agi.set_variable('WAZO_RECORD_GROUP_CALLEE', '1')
+        try:
+            calld.calls.start_record(channel_id)
+        except Exception as e:
+            import logging
```
Comment (bloom1) [inline wazo_agid/handlers/group.py:59]: Is there a reason to import logging here instead at the top of the file?

File: wazo_agid/handlers/queue.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        if not should_record:
+            return
+
+        calld = self._agi.config['calld']['client']
+        channel_id = self._agi.env['agi_uniqueid']
+        try:
+            calld.calls.start_record(channel_id)
+        except Exception as e:
+            import logging
+            logger = logging.getLogger(__name__)
```
Comment (bloom1) [inline wazo_agid/handlers/queue.py:61]: Same question as previous

File: wazo_agid/modules/tests/test_queue.py [hunk 4]
```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
-# Copyright 2013-2019 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2013-2021 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 import unittest
 
-from mock import Mock
+from mock import Mock, patch
```
Comment (bloom1) [inline wazo_agid/modules/tests/test_queue.py:7]: unused import

File: wazo_agid/handlers/group.py (hunk 2 above)
Comment (pc-m) [reply wazo_agid/handlers/group.py:59]: No clue, this is the kind of thing I do when debugging when I want to remove the logger before committing. I moved the import

**Review by bloom1 — APPROVED** [review]

**Pull request merged**